### PR TITLE
adds primeicons #11356

### PR DESCRIPTION
**Pull request by chrabyrd**
### Types of changes

-   [ ] Bugfix (non-breaking change which fixes an issue)
-   [x] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Description of Change

This adds support for primeicons

To test:

1. Checkout this branch
2. In a project, either point to this branch in `package.json` for the arches dependency, or manually the the `primeicons` dependency.
3. Set up a Vue application, or use an already-existing one
4. Render an icon with `<i class="pi pi-${something}></i>`
5. Notice the icon renders

### CODE REVIEW

File: arches/app/templates/base-root.htm [hunk 1]
```diff
@@ -45,6 +45,7 @@
 
     {% block css %}
         <link rel="stylesheet" href="{% webpack_static 'node_modules/font-awesome/css/font-awesome.min.css' %}" />
+        <link rel="stylesheet" href="{% webpack_static 'node_modules/primeicons/primeicons.css' %}" />
```
Comment (jacobtylerwalls) [inline arches/app/templates/base-root.htm:48]: Do we also want these in base.htm? I can imagine the controlled list manager migrating to prime icons.

File: package.json [hunk 2]
```diff
@@ -74,6 +74,7 @@
         "moment": "^2.29.4",
         "nouislider": "11.0.3",
         "numeral": "^2.0.6",
+        "primeicons": "^7.0.0",
```
Comment (jacobtylerwalls) [inline package.json:77]: A little silly, but probably want to keep the 8.0.0.md added dependencies up to date

File: arches/app/templates/base-root.htm (hunk 1 above)
Comment (chrabyrd) [reply arches/app/templates/base-root.htm:48]: Yes, but not directly. `base.htm` defers css to `arches.scss`

File: package.json (hunk 2 above)
Comment (chrabyrd) [reply package.json:77]: oh yeah, ++

**Review by jacobtylerwalls — APPROVED** [review]

**Pull request merged**